tools/grammar-demo/src/main/java/org/opencypher/demo/grammar/AntlrParserDemo.java

```diff
@@ -77,21 +79,28 @@ public static void main( String[] args )
 
     public AntlrParserDemo( boolean useTreeListener, boolean createGrammarFromXML, String query, String indentStep)
     {
-        org.antlr.v4.tool.Grammar grammar = createGrammarFromXML ? createGrammarFromXML( "/cypher.xml" ) : readGrammarFromG4( "/Cypher.g4" );
-        LexerInterpreter lexer = grammar.createLexerInterpreter( CharStreams.fromString( query ) );
-        ParserInterpreter parser = grammar.createParserInterpreter( new CommonTokenStream( lexer ) );
-        lexer.removeErrorListeners();
-        parser.removeErrorListeners();
-        lexer.addErrorListener( new FailingErrorListener( query ) );
-        parser.addErrorListener( new FailingErrorListener( query ) );
-        if( useTreeListener )
-        {
-            parser.addParseListener( new MyParseTreeListener(grammar, indentStep) );
+        org.antlr.v4.tool.Grammar grammar;
+        if(createGrammarFromXML) {
```

Comment:
Does this file use a newline-before-`{` style? If so, this looks off :slightly_smiling_face:

Reply:
Thanks. I fixed the coding style.